scripts/run_vcwallet_ui_automation.sh

```diff
@@ -54,7 +54,7 @@ healthCheck() {
 }
 
 # healthcheck
-healthCheck wallet-web https://wallet.trustbloc.local:8091/healthcheck 200
+healthCheck vcwallet https://vcwallet.trustbloc.local:8071/healthcheck 200
```

Comment:
if we are replacing wallet-web what will be the another instance which will run with chapi and didcomm enabled ?

Reply:
https://github.com/trustbloc/wallet/pull/1871/files#diff-bf6f61dd6a378d5262de4cb22c38fcd438d891d9ecadb9526f625e1461c39d33R9-R61